Frameworks/CoreGraphics/CGFont.mm

```diff
@@ -100,11 +100,17 @@ CGFontRef CGFontRetain(CGFontRef font) {
 }
 
 /**
- @Status Stub
+ @Status Interoperable
 */
 CGFontRef CGFontCreateWithDataProvider(CGDataProviderRef cgDataProvider) {
```

Comment:
returns null

---
In reply to: [88145504](https://github.com/Microsoft/WinObjC/pull/1386#discussion_r88145504) [](ancestors = 88145504)